maintenance_marketing.Rmd

```diff
@@ -10,7 +10,7 @@ We will help you promoting your package but here are some more things to keep in
 
```

Comment:
Should we mention `usethis::use_release_issue()` here? It also helps a lot to smooth the submission process and make sure you didn't forget one of the many checks necessary to get it on CRAN

Reply:
It's in https://devdevguide.netlify.app/releasing.html#releasing-1 but maybe I should add a link to that section?